hack/templates/aws_v1alpha1_configmap.tmpl

```diff
@@ -16,3 +16,8 @@ objects:
     root: ${ROOT}
     base: ${BASE}
     account-limit: ${ACCOUNTLIMIT}
+    MaxReconciles.account: "1"
```

Comment:
We're going to need to parameterize these in hack/olm-registry/olm-artifacts-template.yaml if they're to be settable via app-interface.